services/twitch/twitch.service.js

```diff
@@ -8,39 +8,28 @@ const helixSchema = Joi.object({
 })
 
 module.exports = class TwitchStatus extends TwitchBase {
-  static get category() {
-    return 'activity'
-  }
+  static category = 'activity'
 
-  static get route() {
-    return {
-      base: 'twitch/status',
-      pattern: ':user',
-    }
+  static route = {
+    base: 'twitch/status',
+    pattern: ':user',
   }
 
-  static get examples() {
-    const preview = this.render({
-      user: 'andyonthewings',
-      isLive: true,
-    })
-    // link[] is not allowed in examples
-    delete preview.link
-    return [
-      {
-        title: 'Twitch Status',
-        namedParams: {
-          user: 'andyonthewings',
-        },
-        staticPreview: preview,
```

Comment:
hmm. I guess this okay since there's just this one example, but in general want to be mindful about having the examples be completely disconnected from the actual rendering function due to the potential for drift. I'm sure there's a handful of other cases within the codebase, just a bit worried a hypothetical future change could update the actual render function and not be aware of the need to update the preview

Reply:
@calebcartwright agreed. will keep this in mind. you can assign me in the future if any changes happen.